Makefile

```diff
@@ -158,9 +158,9 @@ requirements_ansible: virtualenv_ansible
 	$(VENV_BASE)/ansible/bin/pip uninstall --yes -r requirements/requirements_ansible_uninstall.txt
 
 requirements_ansible_py3: virtualenv_ansible_py3
-	cat requirements/requirements_ansible.txt requirements/requirements_ansible_git.txt | $(VENV_BASE)/ansible3/bin/pip3 install $(PIP_OPTIONS) --no-binary $(SRC_ONLY_PKGS) --ignore-installed -r /dev/stdin
-	$(VENV_BASE)/ansible3/bin/pip3 install ansible  # can't inherit from system ansible, it's py2
-	$(VENV_BASE)/ansible3/bin/pip3 uninstall --yes -r requirements/requirements_ansible_uninstall.txt
+	cat requirements/requirements_ansible.txt requirements/requirements_ansible_git.txt | $(VENV_BASE)/ansible/bin/pip3 install $(PIP_OPTIONS) --no-binary $(SRC_ONLY_PKGS) --ignore-installed -r /dev/stdin
```

Comment:
In `make requirements_dev`, we call both the py2 *and* py3 target because our dev environment gives you a py2 *and* py3 venv by default:

https://github.com/ansible/awx/pull/3403/files#diff-b67911656ef5d18c4ae36cb6741b7965R192

...meaning *today* a `make docker-compose` will create `/venv/ansible` *and* `/venv/ansible3`

Is this change going to break this/cause conflicts on `$(VENV_BASE)/ansible`

I'm open to other approaches to this for the dev env, I just want to make sure we're not going to break the dev env.

Reply:
@ryanpetrello That's a good point that I hadn't taken into consideration. I didn't realize that the development environment did that. I ran into issues with this and adapting it to work with our downstream packaging, and the solution was to make the paths the same. I'll look into alternative options and see what is possible.